tket/src/Circuit/ThreeQubitConversion.cpp

```diff
@@ -97,6 +97,30 @@ static Circuit two_qubit_diag_adjoint_plex_tk(const Eigen::Matrix4cd &D) {
   return circ;
 }
 
+static const std::vector<Eigen::Matrix4cd> &get_conj_unitaries() {
+  auto make_conj_unitaries = []() -> std::vector<Eigen::Matrix4cd> {
```

Comment:
This could be `static const` too I think.

Reply:
This should be resolved now.